src/allmydata/web/root.py

```diff
@@ -182,9 +183,44 @@ def data_import_path(self, ctx, data):
     def render_my_nodeid(self, ctx, data):
         tubid_s = "TubID: "+self.client.get_long_tubid()
         return T.td(title=tubid_s)[self.client.get_long_nodeid()]
+
     def data_my_nickname(self, ctx, data):
         return self.client.nickname
 
+    def renderHTTP(self, ctx):
+        req = inevow.IRequest(ctx)
+        t = get_arg(req, "t")
```

Comment:
(no action required)

I have started writing string literals using _only_ the `b` or `u` prefixes so that every string is explicit and unambiguous about its type.  I don't know if Tahoe-LAFS has any kind of project policy about such things.  I'd recommend it, though.